openslides/utils/rest_api.py

```diff
@@ -45,7 +45,7 @@ def get_collection_and_id_from_url(url):
     Raises OpenSlidesError if the URL is invalid.
     """
     path = urlparse(url).path
-    match = re.match(r'^/api/(?P<name>[-\w]+/[-\w]+)/(?P<id>[-\w]+)/$', path)
+    match = re.match(r'^/rest/(?P<name>[-\w]+/[-\w]+)/(?P<id>[-\w]+)/$', path)
```

Comment:
Please rename 'name' to 'collection' here and in line 51. I forgot this in my last pull request.